src/osd/osd_types.h

```diff
@@ -3497,7 +3497,7 @@ class PastIntervals {
       old_up_primary, new_up_primary,
       old_up, new_up,
       same_interval_since, last_epoch_clean,
-      osdmap.get(), lastmap.get(),
+      osdmap.load(), lastmap.load(),
```

Comment:
See make check:
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/osd/osd_types.h:3487:15: note: in instantiation of template class 'std::atomic<std::shared_ptr<const OSDMap>>' requested here
    OSDMapRef osdmap,      ///< [in] current map
              ^
/home/jenkins-build/build/workspace/ceph-pull-requests/src/osd/osd_types.h:3500:30: error: no member named 'load' in 'std::atomic<std::shared_ptr<const OSDMap>>'
      osdmap.load(), lastmap.load(),
                     ~~~~~~~ ^
2 errors generated.
```

Reply:
Thanks  for taking a look-- I'm not sure how I missed this in my local build!